fastscanner/tests/test_partitioned_csv_provider.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+from datetime import date, datetime, time, timedelta 
+import pandas as pd
+import pytest
+from unittest.mock import MagicMock, patch
+from fastscanner.pkg.localize import LOCAL_TIMEZONE_STR
+import pytz
+
+tz = pytz.timezone(LOCAL_TIMEZONE_STR)
```

Comment:
pytz is deprecated in our python version. In pandas you can just pass the tz string.